### PR TITLE
Organize imports

### PR DESCRIPTION
**Pull request by black-adder**
Using https://github.com/timothycrosley/isort to automatically organize our imports

### CODE REVIEW

**Comment by yurishkuro** [comment]
@black-adder are you sure we want this? seems like it's not worth the noise

**Comment by black-adder** [comment]
I hate the imports being all over the place. I don't think it's too much noise tbf, just a one time thing and then it's handled automatically from then on.

**Comment by yurishkuro** [comment]
I am not opposed to doing that if you want to fix the PR. But does this tool support some sort of formatting style? E.g. this change is making imports less readable imo:

https://github.com/jaegertracing/jaeger-client-python/pull/63/files#diff-4f42106af9829128b1247b591a945a22L5

**Comment by black-adder** [comment]
ill take a look see

**Comment by yurishkuro** [comment]
Fails in py3 build with
```
ERROR: /home/travis/build/jaegertracing/jaeger-client-python/tests/test_crossdock.py Imports are incorrectly sorted.
```

I think running lint in each Py version is a bad idea as there can be incompatibilities.

File: jaeger_client/metrics/prometheus.py [hunk 1]
```diff
@@ -12,10 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from jaeger_client.metrics import MetricsFactory
 from collections import defaultdict
+
 from prometheus_client import Counter, Gauge
 
+from jaeger_client.metrics import MetricsFactory
```
Comment (yurishkuro) [inline jaeger_client/metrics/prometheus.py]: probably would be better to import `from .`

File: setup.py [hunk 2]
```diff
@@ -63,6 +63,7 @@
             'tchannel>=0.27;python_version<"3"',
             'opentracing_instrumentation>=2,<3',
             'prometheus_client',
+            'isort<4.3',
```
Comment (yurishkuro) [inline setup.py]: why lower bound on minor?

File: tests/test_TUDPTransport.py [hunk 3]
```diff
@@ -14,9 +14,10 @@
 
 from __future__ import absolute_import
 
-import mock
 import unittest
 
+import mock
```
Comment (yurishkuro) [inline tests/test_TUDPTransport.py]: I don't get it. Are there actually some rules it uses? Like built-in modules first? That might explain the break on py3

**Comment by yurishkuro** [comment]
closing for inactivity - given how old this is it would need to be re-done anyway.

**Pull request closed without merging**